README.md

```diff
@@ -11,11 +10,17 @@ This project contains a webserver that connects to the [OpenGliderNet],
 saves the received records to a database for 24 hours and relays all data to
 any connected WebSocket clients.
 
+## Docker Quickstart
```

Comment:
you can add a new subsection using
```
Docker Quickstart
------------------------------------------------------------------------------
```

but IMHO these kinds of instructions belong in a `CONTRIBUTING.md`, not the `README.md`

since the production server is not using docker I wouldn't recommend it as the primary way to do things.

Reply:
Ok, I'll create a CONTRIBUTING.md